### PR TITLE
Fix an undefined variable in [GithubPackageJson]

### LINKED ISSUE #4111: ReferenceError: errorMessages is not defined
Sentry Issue: [SHIELDS-48](https://sentry.io/organizations/shields/issues/1253314686/?referrer=github_integration)

```
ReferenceError: errorMessages is not defined
  File "/home/m/shields/services/github/github-common-fetch.js", line 76, in fetchJsonFromRepo
    errorMessages,
  File "/home/m/shields/services/github/github-package-json.service.js", line 70, in GithubPackageJsonVersion.handle
    const { version } = await fetchJsonFromRepo(this, {
  File "/home/m/shields/core/base-service/base.js", line 378, in Function.invoke
    serviceData = await serviceInstance.handle(
  File "/home/m/shields/core/base-service/base.js", line 418, in Object.handler
    const serviceData = await this.invoke(
  File "/home/m/shields/core/base-service/legacy-request-handler.js", line 243, in null.<anonymous>
    const result = handlerOptions.handler(
...
(5 additional frame(s) were not displayed)
```

### PR DESCRIPTION
**Pull request by paulmelnikow**
Make sure this gets linted in the future.

Fix #4111

### CODE REVIEW

File: services/github/github-common-fetch.js [hunk 1]
```diff
@@ -69,11 +66,12 @@ async function fetchJsonFromRepo(
     const json = serviceInstance._parseJson(buffer)
     return serviceInstance.constructor._validate(json, schema)
   } else {
-    const url = `https://raw.githubusercontent.com/${user}/${repo}/${branch}/${filename}`
     return serviceInstance._requestJson({
       schema,
-      url,
-      errorMessages,
```
Comment (paulmelnikow) [inline services/github/github-common-fetch.js]: This was the bad one ^^

File: .eslintrc.yml [hunk 2]
```diff
@@ -114,7 +118,7 @@ rules:
   '@typescript-eslint/no-var-requires': error
 
   # These should be disabled by eslint-config-prettier, but are not.
-  spaced-comment: 'off'
+  no-extra-semi: 'off'
```
Comment (calebcartwright) [inline .eslintrc.yml]: I'm a little confused by this rule swap here. Do we no longer need `spaced-comment` disabled?

Comment (paulmelnikow) [reply .eslintrc.yml]: Ah! Good question.

Without `no-extra-semi: 'off'` there are a bunch of errors related to things Prettier wants just so.

Without `spaced-comment: 'off'` there are no errors, which makes me think that it's no longer needed.

Comment (calebcartwright) [reply .eslintrc.yml]: Gotcha, thanks!

**Pull request merged**